src/main/java/com/epam/edu/kh/business/dao/record/RecordDaoImpl.java

```diff
@@ -0,0 +1,74 @@
+package com.epam.edu.kh.business.dao.record;
+
+import java.util.List;
+
+import org.hibernate.Criteria;
+import org.hibernate.SessionFactory;
+import org.hibernate.criterion.Projections;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import org.springframework.transaction.annotation.Transactional;
+
+import com.epam.edu.kh.business.entity.Record;
+
+@Component("recordDaoImpl")
+public class RecordDaoImpl implements RecordDao {
+
+    public RecordDaoImpl() {
+
+    }
+
+    @Autowired
+    private SessionFactory sessionFactory;
+
+    @Transactional
+    public final void save(final Record record) {
+        sessionFactory.getCurrentSession().save(record);
+    }
+
+    @Transactional
```

Comment:
You can use @Transactional for the whole class and override on methods if needed (for example ReadOnly true/false)